hedera-node/configuration/dev/application.properties

```diff
@@ -16,11 +16,6 @@ ledger.id=0x03
 blockStream.streamMode=BOTH
 nodes.enableDAB=true
 blockStream.blockFileDir=data/blockStreams
-addressBook.useRosterLifecycle=true
 grpc.nodeOperatorPortEnabled=true
 # For CI tests we want to override roster weights from override-network.json
-networkAdmin.preserveStateWeightsDuringOverride=false
-# (FUTURE) Remove this when roster lifecycle is permanently adopted; used to simplify
-# the transition to the new roster lifecycle by always exporting both config.txt and
-# candidate-roster.json on handling a PREPARE_UPGRADE transaction
-networkAdmin.exportCandidateRoster=true
+networkAdmin.preserveStateWeightsDuringOverride=false
```

Comment:
There are many `application.properties` files in the repo. Should other files be changed as well? Removed from the repo if they are not used?
Performance testing relies on `hedera-node/data/config/application.properties` which has not been maintained for all recent changes.

Reply:
Since this PR changes the **defaults** to,
```
addressBook.useRosterLifecycle=true
networkAdmin.exportCandidateRoster=true
```
removed the overrides from this _dev/application.properties_ that turned them on in CI.

I don't think any other file needs to adapt.